nuget/MacMobileCenterPush.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Azure.Mobile.Push</id>
+    <title>Mobile Center Push</title>
+    <authors>Microsoft Corporation</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Microsoft Visual Studio Mobile Center Push package provides push notification capabilities for your mobile applications</summary>
+    <description>This package contains functionalities to send push notifications to users of your application from our portal</description>
+    <copyright>Copyright (c) Microsoft Corporation. All Rights Reserved.</copyright>
+    <projectUrl>https://aka.ms/telgml</projectUrl>
+    <licenseUrl>https://aka.ms/vbgfx2</licenseUrl>
+    <iconUrl>https://aka.ms/p2wifj</iconUrl>
+    <dependencies>
+        <group targetFramework="portable-net45+wp8+wpa81+win8+MonoAndroid10+MonoTouch10+Xamarin.iOS10">
+            <dependency id="Microsoft.Azure.Mobile" version="$version$" />
+        </group>
+        <group targetFramework="Xamarin.iOS10">
+            <dependency id="Microsoft.Azure.Mobile" version="$version$" />
+        </group>
+        <group targetFramework="MonoAndroid403">
+            <dependency id="Microsoft.Azure.Mobile" version="$version$" />
+            <dependency id="Xamarin.GooglePlayServices.Base" version="42.1001.0"/>
```

Comment:
It actually doesn't, weirdly - but since i deleted something else i can probably get rid of this now